samples/OriginalUriBaseIds.sarif

```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
+  "properties": {
+    "comment": "This sample demonstrates the use of originalUriBaseIds."
+  },
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "SarifSamples"
+        }
+      },
+      "originalUriBaseIds": {

```

Comment:
>originalUriBaseIds [](start = 7, length = 18)

you don't have an example in here that resolves to an absolute URL

Reply:
I added an example (`LOGSROOT` resolving directly to `[file:///C:/logs/`](file:///C:/logs/`)).

---
In reply to: [414814508](https://github.com/microsoft/sarif-tutorials/pull/14#discussion_r414814508) [](ancestors = 414814508)